spring-client-common/src/main/java/io/camunda/commons/http/JsonUtils.java

```diff
@@ -0,0 +1,60 @@
+package io.camunda.commons.http;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+
+public class JsonUtils {
+
+    private JsonUtils() {
+    }
+
+    private static ObjectMapper mapper;
+
+    private static Map<Class<?>, JavaType> searchResultTypeMap = new HashMap<>();
+
+    private static ObjectMapper getObjectMapper() {
+        if (mapper == null) {
+            mapper = new ObjectMapper();
+            mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+            mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
+        }
+        return mapper;
+    }
```

Comment:
I would recommend to pass a simplified object mapper interface like the `JsonMapper` in the zeebe java client: https://github.com/camunda/zeebe/blob/main/clients/java/src/main/java/io/camunda/zeebe/client/api/JsonMapper.java and default to the Jackson `ObjectMapper`.

Reply:
Added in https://github.com/camunda-community-hub/spring-zeebe/pull/510/commits/092480595ac84d59911023f2e79a6545153a36da